docs-src/cloud-context/saml-how-to-configure-with-azure-ad.md

```diff
@@ -0,0 +1,58 @@
+---
+id: saml-how-to-configure-with-azure-ad
+title: How to configure SAML with Azure AD
+sidebar_label: Configure SAML with Azure AD
+description: To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
+tags:
+  - guide-context
+---
+
+If you want to use the general Microsoft login mechanism, you don't need to set up SAML with Azure AD.
+Just select **Continue with Microsoft** on the Temporal Cloud sign-in page.
+
+To use Azure AD as your SAML IdP, create an Azure AD Enterprise application.
+
+1. Sign in to the [Microsoft Azure AD portal](https://portal.azure.com/).
+1. On the home page under **Manage Azure Active Directory**, select **View**.
```

Comment:
```suggestion
1. On the home page, under **Manage Azure Active Directory**, select **View**.
```

Reply:
Also, if you log in and you get directed to the home page, there's no need to say that at the start